### DISCUSSION TITLE
ScriptReference/SocialPlatforms.GameCenter.GameCenterPlatform

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#296] [Notes]
# ScriptReference/SocialPlatforms.GameCenter.GameCenterPlatform

https://docs.unity3d.com/ScriptReference/SocialPlatforms.GameCenter.GameCenterPlatform.html

**Comment by joonjoonjoon** [comment]
I found it hard to find a starting point based on this so here's some code:  

```csharp
_platform = new GameCenterPlatform();  
_platform.localUser.Authenticate(OnAuthenticate);  
GameCenterPlatform.ShowDefaultAchievementCompletionBanner(true);  
_initialized = true;
```
